### ISSUE TITLE
Cannot read property 'hasOwnProperty' of undefined

### ISSUE THREAD

**Issue by poxin13** [inspheric/nova-indicator-field#1]
Default Text or Select field working but not Indicator
```
Text::make('Status')
```
```
Indicator::make('Status')
  ->labels([
    'a' => 'Active',
]),
```

Same error even doing:
```
Indicator::make('Status')->withoutLabels()
```

**Comment by poxin13** [comment]
Also specifying 'colors' was required, my bad.

**Comment by kitbs** [comment]
This has been fixed in v1.2 so that it will fail gracefully if label and/or colours are not defined.
